packages/ui/src/components/w3m-network-button/styles.ts

```diff
@@ -16,8 +16,8 @@ export default css`
     flex-direction: column;
     align-items: center;
     justify-content: center;
-    width: 80px;
-    height: 80px;
+    width: 78px;
```

Comment:
Is there no way we can make these `%` or `em`? just be more responsive just in case

Reply:
I'll try to play with these, to see if I can 👍 but they might look a bit off on wider mobiles, as we stretch there. Perhaps `display: grid` could be good to investigate as well